src/js/button.js

```diff
@@ -26,24 +26,14 @@ class Button extends ClickableComponent {
    * @method createEl
    */
   createEl(tag = 'button', props = {}, attributes = {}) {
+    if (tag !== 'button') {
+      throw new Error('Buttons must be created with a "button" tag.');
```

Comment:
There were still a couple of components which are clickable <div>s instead of <button>s - one of them was the poster image (I don't remember the other). I'd been discouraged from making DOM changes, so I made this split for backward-compatibility. Ultimately (in v6.0?) the `clickable-component` and the `button` components should be merged again, and the option to pass a tag removed.